workflow/templates/general_html_report/index.html

```diff
@@ -0,0 +1,205 @@
+            <td>  {{ pipeline.version }}</td>
+            <td><a href="{{ pipeline.uri }}">{{ pipeline.uri }}</a></td>
+          </tr>
+      </thead>
+      </table>
+        <div class="bottom">
+        <p><small>Report generated at {{ metadata.report_date }}</small></p>
+        <p><small>Sample analysed at {{ metadata.analysis_date }}</small></p>
+        <!-- x-release-please-start-version -->
+        <!-- x-release-please-end-version -->
+      </div>
+    </div>
+
+        {% for k in nav_header %}
+          <div id = "{{ k }}" class="tabcontent">
+            <h1>{{ pipeline.name }} General report</h1>
+            <h2> {{ k }} results</h2>
+
+          {% for r in results %}
+            {% if r.nav_header == k %}
+              <div class="padding">
+              <hr>
+              <h3>{{ r.name }}</h3>
+              <p>{{ r.description }}</p>
+              {% if r.type == "table" %}
+              <table>
+                <thead>
+                  <tr>
+                    {% for c in r.value[0].keys() %}
+                    <th>{{ c }}</th>
+                    {% endfor %}
+                  </tr>
+                </thead>
+                <tbody>
+                  {% for row in r.value %}
+                  <tr>
+                    {% for k in r.value[0].keys() %}
+                    <td>{{ row[k] }}</td>
+                    {% endfor %}
+                  </tr>
+                  {% endfor %}
+                </tbody>
+              </table>
+
+              {% elif r.type == "file_link" %}
+              <a href="{{ r.value}}">{{ r.value }} </a>
+
+              {% elif r.type == "single_value" %}
+              <div class="single">
+              <p>{{ r.value }}</p>
+              </div>
+              {% elif r.type == "plain_text" %}
+              <div class="single">
+              <p>{{ r.value }}</p>
+              </div>
+              {% elif r.type == "image" %}
+              <img src="{{ r.value }}" />
+
+              {% elif r.type == "file_table" %}
+              <div class="scroll">
+              {{ r.data }}
+              </div>
+              {% elif r.type == "multiqc" %}
+              {% for t in r.data %}
+              {% if "table" in t %}
+              <div class="scroll">
+
+
+              <table>
+                <thead>
+                  <tr>
+                    <th title="Sample name">Sample</th>
+                    {% for k, v in t["table"]["header"].items() %}
+                    <th title="{{ v.description }}">{{ v.title }}</th>
+                    {% endfor %}
+                  </tr>
+                </thead>
+                <tbody>
+                  {% for k, v in t["table"]["data"].items() %}
+                  <tr>
+                    <td>{{ k }}</td>
+                    {% for hk in t["table"]["header"].keys() %}
+
+                    <td style="background-color: rgba({{t["table"]["header"][hk]["colour"]}}, 0.2)" >{{ v[hk] }} </td>
```

Comment:
The quoting here might be an issue, but again, that would need a MultiQC test case.

Reply:
Not a problem as far as I can tell.